lib/chef/provisioning/transport/winrm.rb

```diff
@@ -36,8 +36,11 @@ def initialize(endpoint, type, options, global_config)
       def execute(command, execute_options = {})
         output = with_execute_timeout(execute_options) do
           session.set_timeout(execute_timeout(execute_options))
-          session.run_powershell_script(command) do |stdout, stderr|
-            stream_chunk(execute_options, stdout, stderr)
+          block = Proc.new { |stdout, stderr| stream_chunk(execute_options, stdout, stderr) }
+          if execute_options[:raw]
```

Comment:
Do you think there will be a 3rd way to run Windows commands in the future?  I've been converting a lot of Boolean flags to Symbols recently (to support more than 2 binary options) and if that will happen with this I would rather use a symbol flag now.